### PR TITLE
Lesson007

### PR DESCRIPTION
**Pull request by DenegaSergey**
Add Task Lesson007

### CODE REVIEW

File: Lesson007/Task 1 HW07.StringWorkerTwo/Program.cs [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                        arreyWords[i] = arreyWords[i + 1];
+                    }
+                    Array.Resize(ref arreyWords, arreyWords.Length - 1);
+                    Console.WriteLine(String.Join(" ", arreyWords) + "\n");
+                }
+            }
+
+            {
+                // количество пунктуации
+                int punctuationCount = 0;
```
Comment (Onix-one) [inline Lesson007/Task 1 HW07.StringWorkerTwo/Program.cs]: need to delete unused comments

**Review by Onix-one — CHANGES_REQUESTED** [review]

File: Lesson007/Task 3 HW07.Pyramid/Program.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                    arrey[2, 2] = 1;
+                    arrey[2, 3] = 1;
+                    arrey[3, 0] = 1;
+                    arrey[3, 1] = 1;
+                    arrey[3, 2] = 1;
+                    arrey[3, 3] = 1;
+                    arrey[3, 4] = 1;
+
+                    Console.Write(arrey[y, x] + "\t");
+                }
```
Comment (Onix-one) [inline Lesson007/Task 3 HW07.Pyramid/Program.cs]: need to rewrite this logic. its equal that we do this =>
Console.Write("H");
Console.Write("e");
Console.Write("l");
Console.Write("l");
Console.Write("o");
Console.Write(" ");
Console.Write("w");
Console.Write("o");
Console.Write("r");
Console.Write("l");
Console.Write("d");

**Review by Onix-one — CHANGES_REQUESTED** [review]

File: Lesson007/Task 5 HW07.BookStore/Program.cs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                {
+                   {29, 27, 77, 14},
+                   {78, 03, 55, 17},
+                   {48, 19, 68, 80},
+                   {68, 41, 14, 07}
+                }
+            };
+
+            int max = arreyBooks[0, 0, 0], min = arreyBooks[0, 0, 0];
+
```
Comment (Onix-one) [inline Lesson007/Task 5 HW07.BookStore/Program.cs]: usually need to use int.MaxValue as max value and int.MinValue as min value

**Review by Onix-one — CHANGES_REQUESTED** [review]

File: Lesson007/Task 5 HW07.BookStore/Program.cs (hunk 3 above)
Comment (DenegaSergey) [reply Lesson007/Task 5 HW07.BookStore/Program.cs]: Fixed

File: Lesson007/Task 1 HW07.StringWorkerTwo/Program.cs (hunk 1 above)
Comment (DenegaSergey) [reply Lesson007/Task 1 HW07.StringWorkerTwo/Program.cs]: Fixed

File: Lesson007/Task 3 HW07.Pyramid/Program.cs (hunk 2 above)
Comment (DenegaSergey) [reply Lesson007/Task 3 HW07.Pyramid/Program.cs]: Fixed))

**Pull request merged**